cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -128,12 +128,22 @@ function(_omr_toolchain_separate_debug_symbols tgt)
 		omr_get_target_output_genex(${tgt} output_name)
 		set(dbg_file "${output_name}${OMR_DEBUG_INFO_OUTPUT_EXTENSION}")
 		if(OMR_OS_AIX AND CMAKE_C_COMPILER_IS_OPENXL)
-			add_custom_command(
-				TARGET "${tgt}"
-				POST_BUILD
-				COMMAND "${CMAKE_COMMAND}" -E copy ${exe_file} ${dbg_file}
-				COMMAND "${CMAKE_STRIP}" -X32_64 ${exe_file}
-			)
+			# When building with OpenXL on AIX, get rid of strip for libjvmtitest.so until
+			# https://github.com/eclipse-openj9/openj9/issues/21528 is resolved.
+			if ("${tgt}" MATCHES "${SKIP_STRIP_MODULE}")
```

Comment:
I like the idea of adding a strip-or-not configuration point, but this would make it awkward to skip more than one "target" (which I suggest is a more appropriate name than "module").

I think we should use a target property (see [OmrTargetSupport.cmake](https://github.com/eclipse-omr/omr/blob/master/cmake/modules/OmrTargetSupport.cmake#L207)). Perhaps it would be sufficient to add this in a suitable place in openj9:
```
    # Avoid stripping jvmtitest by pretending it has already been done.
	set_target_properties("jvmtitest" PROPERTIES OMR_SPLIT_DEBUG_PROCESSED TRUE)
```
Then we need no change to omr at all.

Reply:
Can we skip calling `_omr_toolchain_separate_debug_symbols` for jvmtitests (or any other targets we may not want to strip in the future) entirely? Aside from the `strip` step, there's also this step that's used to create the `.debuginfo` file: 
https://github.com/eclipse-omr/omr/blob/46f99d1c7d43bc7d05d31791aafe694201c6a1cc/cmake/modules/platform/toolcfg/gnu.cmake#L143
Of course, if we're skipping the strip step, the `.debuginfo` file's contents would be identical to that of its corresponding shared library. My concern is that there may be situations where the `.debuginfo` file specifically is needed/expected, so I just want to confirm that this won't be the case before proceeding with this approach.

Reply:
@keithc-ca I came up with a new solution that uses target properties instead of a normal variable. Part of that solution involves some changes on the OpenJ9 side, which I've opened a PR for here: https://github.com/eclipse-openj9/openj9/pull/21689

Please feel free to review when you have a moment!